services/QuillLMS/spec/queries/teachers_data_spec.rb

```diff
@@ -40,8 +40,8 @@
 
   let!(:concept1) { create(:concept) }
   let!(:concept2) { create(:concept) }
-  let!(:concept_result1) { create(:old_concept_result, concept: concept1, activity_session: activity_session1) }
-  let!(:concept_result2) { create(:old_concept_result, concept: concept2, activity_session: activity_session2) }
+  let!(:concept_result1) { create(:concept_result, concept: concept1, activity_session: activity_session1) }
```

Comment:
I'm surprised this test works by changing these from `old` to `new` concept_results. I'm guessing these aren't used.

Reply:
You were right.  I'm removing these lines.